etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -141,7 +141,11 @@ func (s *simplePrinter) RoleGet(role string, r v3.AuthRoleGetResponse) {
 	printRange := func(perm *v3.Permission) {
 		sKey := string(perm.Key)
 		sRangeEnd := string(perm.RangeEnd)
-		fmt.Printf("\t[%s, %s)", sKey, sRangeEnd)
+		if strings.Compare(sRangeEnd, "\xff") != 0 {
```

Comment:
Sorry for my confusing writing. Yes, I'll just add the checking against "\x00" here. The idea of the affine stuff will be implemented in the perm cache side. My comment wasn't limited to the above line.